src/modules/endpoint/DatabaseConnector.js

```diff
@@ -339,6 +352,46 @@ class DatabaseConnector {
     const path = this.getContentPath('languages', context)
     await this.writeFile(path, JSON.stringify(languages))
   }
+  
+  async storePois (pois: Array<PoiModel>, context: DatabaseContext) {
+    const jsonModels = pois.map((poi: PoiModel): ContentPoiJsonType => ({
+      path: poi.path,
+      title: poi.title,
+      content: poi.content,
+      thumbnail: poi.thumbnail,
+      availableLanguages: poi.availableLanguages,
+      excerpt: poi.excerpt,
+      location: poi.location,
+      lastUpdate: poi.lastUpdate,
+      hash: poi.hash
+    }))
+    await this.writeFile(this.getContentPath('pois', context), JSON.stringify(jsonModels))
+  }
+
+  async loadPois (context: DatabaseContext): Promise<Array<PoiModel>> {
+    const path = this.getContentPath('pois', context)
+    const fileExists: boolean = await RNFetchBlob.fs.exists(path)
+
+    if (!fileExists) {
+      throw Error(`File ${path} does not exist`)
+    }
+
+    const json = JSON.parse(await this.readFile(path))
+
+    return json.map((jsonObject: ContentPoiJsonType) => {
+      return new PoiModel({
+        path: jsonObject.path,
+        title: jsonObject.title,
+        content: jsonObject.content,
+        thumbnail: jsonObject.thumbnail,
+        availableLanguages: jsonObject.availableLanguages,
+        excerpt: jsonObject.excerpt,
+        location: jsonObject.location,
+        lastUpdate: jsonObject.lastUpdate,
```

Comment:
`lastUpdate: moment(jsonObject.last_update, moment.ISO_8601),`

Please check that load/store code actually works. You can do this using `adb shell`. Then you can use cat to see the jsons: `cat /data/data/de.tuerantuer.integreat/files/cities.json`